workflow/rules/common.smk

```diff
@@ -1057,3 +1058,19 @@ def get_delly_excluded_regions():
         )
     else:
         return []
+
+
+def get_itd_regions(wildcards):
+    group=samples.loc[wildcards.sample]["group"]
+    return  f"results/regions/{group}.expanded_regions.filtered.bed"
```

Comment:
This can be replaced by
`collect("results/regions/{group}.expanded_regions.filtered.bed", group=lookup(query="sample == '{sample}'", cols="group"))`

Reply:
Which can then be done within the rule if it is only needed right there.